### ISSUE TITLE
Inert gas compounds

### ISSUE THREAD

**Issue by PrakhyaDeepu** [Griffintaur/Periodic2048#20]
But there are few compounds that can be formed when inert gases react with other elements.
Some examples are XeF4 (Xenon Tetraflouride) ,RnF2 (Radon DiFluoride) , Xenon oxide (XeO2) etc. So we cannot entirely rule out that inert gases never react. Instead we can show few basic structures that are formed.

**Comment by Griffintaur** [comment]
@PrakhyaDeepu Agreed. But I believe we should focus on the general structure rather than exceptions as we all know chemistry is full of exceptions.
